docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,195 @@
+
+       NAME                                           READY   UP-TO-DATE   AVAILABLE   AGE
+       deployment.apps/my-trino-cluster-coordinator   1/1     1            1           161m
+       deployment.apps/my-trino-cluster-worker        2/2     2            2           161m
+
+       NAME                                                     DESIRED   CURRENT   READY   AGE
+       replicaset.apps/my-trino-cluster-coordinator-bfb74c98d   1         1         1       161m
+       replicaset.apps/my-trino-cluster-worker-76f6bf54d6       2         2         2       161m
+
+Executing queries
+-----------------
+
+The pods running the Trino containers are all running on a private network
+internal to Kubernetes. In order to access them, specifically the coordinator,
+you will need to create a tunnel to the exposed service and your computer. You
+can do this by running the commands generated upon installation.
+
+#. Store the coordinator pod name in a shell variable called ``POD_NAME``.
+
+   .. code-block:: text
+
+       export POD_NAME=$(kubectl get pods --namespace default -l "app=trino,release=my-trino-cluster,component=coordinator" -o jsonpath="{.items[0].metadata.name}")
+
+#. Create the tunnel from the service to the client. After you run this you can
+   connect to the Trino coordinator at ``http://localhost:8080``.
+
+   .. code-block:: text
+
+       kubectl port-forward $POD_NAME 8080:8080
+
+#. To connect to Trino, you can use the :doc:`/client/cli` or your favorite
+   client that supports using a :doc:`/client/jdbc`.
+   :ref:`Install the command-line interface <cli-installation>`, and connect to
+   Trino.
+
+   .. code-block:: text
+
+       ./trino --server http://localhost:8080
+
+#. Using the generated benchmark data in the ``tpch`` catalog, type and execute
+   a query on the ``nation`` table using the scale factor schema, ``sf1``. Try
+   other commands to explore this data set:
+
+   .. code-block:: text
+
+       $ trino> select count(*) from tpch.sf1.nation;
+        _col0
+       -------
+         25
+       (1 row)
+
+       Query 20181105_001601_00002_e6r6y, FINISHED, 1 node
+       Splits: 21 total, 21 done (100.00%)
+       0:06 [25 rows, 0B] [4 rows/s, 0B/s]
+
+#. Once you are done with your exploration, enter the ``quit`` command.
+
+
+Configuring Helm for Trino
+--------------------------
+
+The :doc:`container image </installation/containers>` already contains a
+default configuration to get started, and some catalogs to allow you to explore
+Trino. You can customize these configuration by updating the ``values.yaml``
+file.
+
+
+The ``values.yaml`` has various properties that, when not empty, will create
+configuration files in the ``/etc/trino`` directory structure as created in the
+:doc:`/installation/deployment`. One common configuration request is to add
+custom catalogs to the kubernetes deployment. You can do this by adding values
+to the ``additionalCatalogs`` property in the ``values.yaml`` file.
```

Comment:
I want to do a much fuller list of all the properties. I'm struggling about where we do it thought. We have a list of properties on the [Trino Charts GitHub](https://github.com/trinodb/charts/tree/main/charts/trino#configuration) which propagates to the readme seen on the [Trino ArtifactHub page](https://artifacthub.io/packages/helm/trino/trino). I kind of want to add the descriptions there and then link to that readme from these docs. WDYT @mosabua?

Reply:
This page should not be the reference, the Chart readme should. I'd add just one more section, about configuring the number of Trino nodes, and their resource requests and limits.

Reply:
Agreed ... but we can and should also help on the chart repo and in fact from there link to this page once its live..

Reply:
Cool so we all agree that we maintain the properties there and that we should link to those properties from this document. I like that a lot as it maintains a single location and is visible from GitHub and ArfifactHub.